src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v2-monitoring-integration.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Amazon API Gateway V2 Monitoring Integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS API Gateway v2 monitoring integration: what data it reports, and how to enable it."
+freshnessValidatedDate: 2023-07-10
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) now include an integration for AWS API Gateway V2, sending its metrics data to New Relic.
```

Comment:
@rahul188 I would make the "V2" lower case as you have it in the metaDescription: v1

Reply:
@rhetoric101, I made the suggested change.